packages/crud/src/vaadin-crud-mixin.js

```diff
@@ -400,7 +400,7 @@ export const CrudMixin = (superClass) =>
       this.addController(this._gridController);
 
       // Init button controllers in `ready()` (not constructor) so that Flow can set `_noDefaultButtons`
-      this._newButtonController = new ButtonSlotController(this, 'new', 'primary', this._noDefaultButtons);
+      this._newButtonController = new ButtonSlotController(this, 'new', null, this._noDefaultButtons);
```

Comment:
This is a minor breaking change, I'll make a separate ticket and a new PR for it.

Reply:
Created https://github.com/vaadin/web-components/pull/9817, will rebase this PR on top of it.